test/helpers/test.go

```diff
@@ -319,6 +319,10 @@ func (t *TestOptions) TheApplicationIsRunning(statusCode int, environment string
 		if u == "" {
 			return fmt.Errorf("no URL found for environment %s has app: %#v", environment, applications)
 		}
+		if u == "1/1" {
```

Comment:
-1, imo this is just covering up an underlying problem somewhere, most likely in `parsers.ParseJxGetApplications`. My guess is that the output of `jx get application` varies and `ParseJxGetApplications` is not handling this properly. "1/1" is probably the pod count and should never end up in the URL field of the application struct. 

My guess is that `Url:         fields[len(fields)-1],` causes the issue. Might be that the pod is running, but the ingress not exposed yet. Just choosing the last field of the output seems wrong. At least it should be validated.

For what it's worth using `jx get application` in this helper is questionable to start with.

Reply:
So my intent here is really to try to debug what exactly is going wrong. Currently, when we hit this case, we don’t log the `jx get applications` output, so there’s no way to tell exactly what’s being parsed in the first place. I wanted to add that logging, and just in case it turns out to be something that clears up on a second run, retry. I’m happy to get rid of the retry, but we definitely need the logging to figure out what’s breaking.

Reply:
so why not add this to `parsers.ParseJxGetApplications`. That's where it goes wrong right? Let's try to fix and debug things at the source of the problem. `u == "1/1"` is wrong on so many levels.

Reply:
Because I literally didn't think of that! Whoops. Doing so now.